controllers/numaresourcesoperator_controller.go

```diff
@@ -73,6 +73,12 @@ import (
 
 const numaResourcesRetryPeriod = 1 * time.Minute
 
+// MCPDaemonSetPair a struct to hold the target MCP of a configured node group and its created respective RTE daemonset
+type MCPDaemonSetPair struct {
+	MCPName   string
+	Daemonset nropv1.NamespacedName
+}
+
```

Comment:
if we call this
```
type poolDaemonSet struct {
    PoolName string
    DaemonSet nropv1.NamespacedName
}
```
we generalize it for free.
In addition, this type is (should be) used only internally, so should not be exported (type should start lowercase).

Reply:
right, I'll update that. Thanks for catching it.